clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -83,10 +86,14 @@
    * @param options the client options
    */
   public FileInStream(FileInfo info, InStreamOptions options) {
+    this(info, options, FileSystemContext.INSTANCE);
+  }
+
+  FileInStream(FileInfo info, InStreamOptions options, FileSystemContext context) {
```

Comment:
I don't see any way to let tests control the FileSystemContext without either doing this or making mContext non-final. Any ideas @jsimsa?